util/GO9PTERM

```diff
@@ -16,8 +16,11 @@ if [ "$(uname)" = "Linux" ] && [ -e /dev/kvm ]; then
 	fi
 fi
 
+# specific usb devices added by '-device usb-host,hostbus=X,hostaddr=X'
+# replacing 'X' with the device numbers provided by a tool like 'lsusb'
 read -r cmd <<EOF
 $kvmdo qemu-system-x86_64 -s -cpu Opteron_G1 -smp 2 -m 2048 $kvmflag \
+-usb \
```

Comment:
Line 19 is the usb device example. You want lsusb output examples in the file as well? There is no guarantee what address a given usb device will receive from what I can tell.
I can't put comments in the qemu command because I'm not sure how to escape the backslashes.